manifests/candlepin.pp

```diff
@@ -48,14 +47,6 @@
     enable_basic_auth            => false,
     consumer_system_name_pattern => '.+',
     adapter_module               => 'org.candlepin.katello.KatelloModule',
-    amq_enable                   => true,
-    amqp_keystore_password       => $certs::candlepin::keystore_password,
-    amqp_truststore_password     => $certs::candlepin::keystore_password,
-    amqp_keystore                => $certs::candlepin::amqp_keystore,
-    amqp_truststore              => $certs::candlepin::amqp_truststore,
-    qpid_hostname                => $katello::params::qpid_hostname,
-    qpid_ssl_cert                => $certs::candlepin::client_cert,
-    qpid_ssl_key                 => $certs::candlepin::client_key,
```

Comment:
I'll need you to add passing a new parameter:

```
artemis_client_dn => $artemis_client_dn
```

And then before the `class { 'candlepin`:` declration:

``
$artemis_client_dn = "CN=${certs::candlepin::cname}, OU=${certs::candlepin::org_unit}, O=candlepin, L=${certs::candlepin::country}, ST=${certs::candlepin::state}, C=${certs::candlepin::city}"
```

Reply:
@ehelms  what do you think about defining `$artemis_client_dn` in certs and here we will do:

`artemis_client_dn => $certs::candlepin::artemis_client_dn` ?

Reply:
Done. You can now update to reference it.